tasks.py

```diff
@@ -25,6 +51,27 @@ def run_test_cmd(ctx, cmd, env=None) -> int:
     return ctx.run(cmd, warn=True, env=env).exited
 
 
+@task
+def watch_tests(ctx):
```

Comment:
`poetry run invoke watch-tests` while developing and you get tests automatically run every time you save .. and because this stupid observer sends every event separately a few times more every now and then, but that's still better than manual effort.